doc/input-output-reference/src/overview/group-refrigeration.tex

```diff
@@ -1055,7 +1055,7 @@ \subsubsection{Field: Insulated Floor U-Value}\label{field-insulated-floor-u-val
 
 The floor themal transmittance (in W/m\(^{2}\)-K). This value has a default value of 0.3154. (This corresponds to R18 in Archaic American Insulation Units.~ To convert other R-values to thermal transmittance, divide 5.678 by the R-value.~ For example, R15 is 0.3785 W/m\(^{2}\)-K and R5 is 1.136 W/m\(^{2}\)-K.)
 
-\textbf{\emph{THE REMAINING 12 FIELDS FOR THE WALK-IN COOLER MUST BE REPEATED FOR EACH ZONE WHICH IS IN CONTACT WITH A WALK-IN WALL, CEILING, OR DOOR. The IDD includes fields for 3 zones, but can be extended by repeating the last 12 values in the object.}}
+\textbf{\emph{THE REMAINING 12 FIELDS FOR THE WALK-IN COOLER MUST BE REPEATED FOR EACH ZONE WHICH IS IN CONTACT WITH A WALK-IN WALL, CEILING, OR DOOR.  This field is extensible, so additional fields of this type can be added to the end of this object by repeating the last 12 values in the object.}}
```

Comment:
This *object* is extensible.
Actually, most of the places that say "This field is extensible..." probably should say "This object is extensible" I guess saying "this field" is ok for places where it's just a single extensible field.

Reply:
Generally my pattern was if it was a single field I used the term field.  If it was a group of fields that were repeated, I adjusted the language to say that.  Are you ok with that?  If not, I'll go through and change them.

Reply:
I'm in the process of changing everything to "This object is extensible..." and cleaning up how these some of these show up in the docs.  So ignore the question from the above comment.